src/models/flinkStatement.ts

```diff
@@ -140,6 +157,23 @@ export class FlinkStatement implements IResourceBase, IdItem, ISearchable, IEnvP
     return this.metadata?.updated_at;
   }
 
+  /**
+   * Is this statement's `updatedAt` newer than other's?
+   * @throws Error if statements have different name or environmentIds.
+   */
+  isFresherThan(other: FlinkStatement): boolean {
```

Comment:
Nit: rename to `isNewerThan` to match the comment

Reply:
Or `isUpdatedMoreRecentlyThan` if we also need to check `isNewerThan` for the `createdAt` property

Reply:
Am gonna respell the comment. If I were to see `isNewerThan`, I'd think that it was doing createdAt comparison. At least having a slightly oddball name makes you wonder a little more.